cmd/postgres_exporter/postgres_exporter.go

```diff
@@ -1841,6 +1853,23 @@ func main() {
 
 	prometheus.MustRegister(exporter)
 
+	registries := make(map[string]*prometheus.Registry)
+	http.HandleFunc("/scrape", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I think we're standardizing on the `/probe` path.

Reply:
Fixed